Zend/zend_attributes.c

```diff
@@ -35,31 +35,41 @@ static zend_object_handlers attributes_object_handlers_sensitive_parameter_value
 static HashTable internal_attributes;
 
 void validate_attribute(zend_attribute *attr, uint32_t target, zend_class_entry *scope)
+{
+}
```

Comment:
This could be removed now.

Reply:
I'm slightly concerned about extensions assuming that the pointer is available. Hence, I'd prefer keeping it for release branches.

Reply:
Will you drop it on the merge to master? Otherwise perhaps add a comment with the response you gave here.

Reply:
I dropped the empty function on master.